spec/controllers/divisions_controller_spec.rb

```diff
@@ -1,5 +1,76 @@
 require 'spec_helper'
 
 describe DivisionsController, :type => :controller do
+  describe "#index" do
```

Comment:
Looking good ⭐️ It's great to start having this crazy action tested. I've added some suggestions. See what you think.

With those tests in place I'd be happy to merge this.